lib/tapioca/dsl/compilers/active_support_current_attributes.rb

```diff
@@ -40,23 +40,29 @@ module Compilers
       # # typed: true
       #
       # class Current
-      #   sig { returns(T.untyped) }
-      #   def self.account; end
+      #   include GeneratedAttributeMethods
       #
-      #   sig { returns(T.untyped) }
-      #   def account; end
+      #   class << self
```

Comment:
Previously the comment was incorrectly showing the class methods being defined with `def self.`, but they are actually defined within `class << self`.